tss/common/utils.go

```diff
@@ -8,6 +8,7 @@ import (
 	"time"
 
 	"github.com/btcsuite/btcd/btcec"
+
```

Comment:
should delete this line

Reply:
This conforms to the specification of our go format imports